openedx/core/djangoapps/course_groups/models.py

```diff
@@ -129,7 +131,25 @@ def assign(cls, cohort, user):
 
     def save(self, force_insert=False, force_update=False, using=None, update_fields=None):
         self.full_clean(validate_unique=False)
-
+        if self.pk is None:
+            COHORT_MEMBERSHIP_CHANGED.send_event(
+                cohort=CohortData(
+                    user=StudentData(
+                        username=self.user.username,
+                        email=self.user.email,
+                        first_name=self.user.first_name,
+                        last_name=self.user.last_name,
+                        is_active=self.user.is_active,
+                        profile=UserProfileData(
+                            meta=self.user.profile.meta,
+                            name=self.user.profile.name,
+                        )
+                    ),
+                    course=CourseOverviewData(
+                        course_key=self.course_id,
+                    ),
+                )
+            )
```

Comment:
is this what we mean _cohort changed_? o do we mean just when users are assigned to a cohort? @felipemontoya